include/crow/task_timer.h

```diff
@@ -61,6 +61,7 @@ namespace crow
             /// Schedule the given task to be executed after the given time.
 
             ///
+            /// \param task .
```

Comment:
This is kind of what I'm talking about, `task` is pretty self explanatory (especially with the developer knowing what `task_type` is). Yet CI will fail if this line isn't added, which is in my opinion adds more tedium than organization.

Reply:
Yes, this was the 1 error that I had to work around.  I have no idea what `task` is.